providers/mount.rb

```diff
@@ -81,11 +81,11 @@ def basic_mount_options
 end
 
 def mount_options_for_backup_server
-  case new_resource.backup_server.class
+  case new_resource.backup_server
   when String
     ',backupvolfile-server=' + new_resource.backup_server
   when Array
-    ',backupvolfile-server=' + new_resource.backup_server.join(',backupvolfile-server=')
+    ',backup-volfile-servers=' + new_resource.backup_server.join(':')
```

Comment:
Could be a platform specific addition?  man mount.glusterfs (glusterfs-fuse-3.7.15-1.el7.x86_64) has the following parameter:

      backup-volfile-servers=SERVERLIST
              Provide list of backup volfile servers in the following format [default: None]

Should I remove this?